plugin/main_test.go

```diff
@@ -12,7 +12,10 @@ import (
 var mainHelper *testlib.MainHelper
 
 func TestMain(m *testing.M) {
-	mainHelper = testlib.NewMainHelperWithOptions(nil)
+	var options = testlib.HelperOptions{
```

Comment:
just for my curiosity, what's the convention that MM follows for `var` vs `:=`? or is it just personal preference?

i've seen `var` being used usually when you're just declaring a variable and not initializing it, and `:=` when you want to declare and initialize.

Reply:
Agreed with your interpretation of when to `:=`. In this specific case, I confess to just copying the pattern used in all other `main_test.go` files. I may go back and unify them all as a separate PR.

Reply:
👍